_includes/tutorials/connect-add-key-to-source/ksql/code/src/statements.sql

```diff
@@ -0,0 +1,15 @@
+CREATE SOURCE CONNECTOR JDBC_SOURCE_POSTGRES_01 WITH (
+    'connector.class'= 'io.confluent.connect.jdbc.JdbcSourceConnector',
+    'connection.url'= 'jdbc:postgresql://postgres:5432/postgres',
+    'connection.user'= 'postgres',
+    'connection.password'= 'postgres',
+    'mode'= 'incrementing',
+    'incrementing.column.name'= 'city_id',
+    'topic.prefix'= 'postgres_',
+    'transforms'= 'copyFieldToKey,extractValuefromStruct',
+    'transforms.copyFieldToKey.type'= 'org.apache.kafka.connect.transforms.ValueToKey',
+    'transforms.copyFieldToKey.fields'= 'city_id',
+    'transforms.extractValuefromStruct.type'= 'org.apache.kafka.connect.transforms.ExtractField$Key',
+    'transforms.extractValuefromStruct.field'= 'city_id',
+    'key.converter' = 'org.apache.kafka.connect.converters.IntegerConverter'
+);
```

Comment:
The statements sent to "production" should also file also include the cities table, too, right?

```suggestion
);

CREATE TABLE CITIES (ROWKEY INT KEY) WITH (KAFKA_TOPIC='postgres_cities', VALUE_FORMAT='AVRO'); 

```

Reply:
Debatable - the tutorial is about adding a key to ingested data, and the production step does just that. Happy to add the table declaration in though if you'd prefer?

Reply:
Yeah, as-is, it does what it says on the tin. I'm fine with leaving out the create table statement.